pkg/tidb/forwarder.go

```diff
@@ -17,18 +17,20 @@ import (
 	"context"
 	"fmt"
 	"net"
+	"strings"
 	"time"
 
 	"github.com/cenkalti/backoff/v4"
 	"github.com/pingcap/log"
 	clientv3 "go.etcd.io/etcd/client/v3"
 	"go.uber.org/fx"
 
+	"github.com/pingcap/tidb-dashboard/pkg/utils/distro"
 	"github.com/pingcap/tidb-dashboard/pkg/utils/topology"
 )
 
 var (
-	ErrNoAliveTiDB = ErrNS.NewType("no_alive_tidb")
```

Comment:
This is an example, in the FE, we use `no_alive_tidb` as the translation key, not value. If we replace it, the FE translation will not work.